pkg/security/podsecuritypolicy/factory.go

```diff
@@ -51,12 +51,6 @@ func (f *simpleStrategyFactory) CreateStrategies(psp *policy.PodSecurityPolicy,
 	}
 
 	var groupStrat group.GroupStrategy
-	if utilfeature.DefaultFeatureGate.Enabled(features.RunAsGroup) {
```

Comment:
this is always true now. So you need to keep what's inside the `if` body

Reply:
done